src/package.ps1

```diff
@@ -96,6 +96,30 @@ function AsPackage {
 	throw "failed to parse package: $Pkg"
 }
 
+function TryEachPackage {
+	param (
+		[Parameter(Mandatory, Position = 0)]
+		[string[]]$Packages,
+		[Parameter(Mandatory, Position = 1)]
+		[scriptblock]$ScriptBlock,
+		[string]$ActionDescription = 'process'
+	)
+	$results = @()
+	$failedPackages = @()
```

Comment:
No uppercase letters in local variables. Is there a way to add a rule to check this?
```suggestion
	$failedpkgs = @()
```

Reply:
I went with `$failures`, since it seemed like most of the other local variables seem to be a single word.